### DISCUSSION TITLE
Fuzzyfind commands

### DISCUSSION THREAD

**Discussion by ingur** [folke/noice.nvim#391] [Q&A] [answered]
Is there a way to use a fuzzy finder for the commandline? I remember wilder.nvim having such a feature but is it possible to have this in the Noice commandline as well?

**Comment by folke** [comment, marked as answer]
Some people use cmp-cmdline with Noice

**Reply by LintaoAmons** [reply]
How to use `arrow keys` instead of `tab` to select the item of the command auto-complete menu?
